Changes

```diff
@@ -53,6 +53,10 @@ Working version
    (Jacques-Henri Jourdan, review by Stephen Dolan and Gabriel Scherer)
 
 ### Standard library:
+- #8771: Lexing: add set_position and set_filename to shift (fake)
```

Comment:
The word "shift" is a bit confusing here (at least to me). "set" isn't accurate?

Reply:
(Or perhaps "to change the initial tracking position of the lexbuf"?)

Reply:
Thanks for review, fixed!

Are the comments in lexing.mli clear? There should be no confusion, that we are not starting actual tokenization from a specific position of the file/string, but we essentially _fake_ the position, which will be inserted in to parse errors/AST nodes.

Reply:
Everything does seem clear enough now, yes.